File: exemplo_integracao_lua/go/exemplo.go
// Exemplo Go usando gopher-lua
// Instale: go get github.com/yuin/gopher-lua
package main
import (
	"github.com/yuin/gopher-lua"
)

func main() {
	L := lua.NewState()
	defer L.Close()
	L.DoString(`print('Olá do Lua rodando no Go!')`)
}

File: exemplo_integracao_lua/rust/exemplo.rs
// Exemplo Rust usando rlua
// Adicione ao Cargo.toml: rlua = "*"
use rlua::Lua;

fn main() {
    let lua = Lua::new();
    lua.context(|ctx| {
        ctx.load("print('Olá do Lua rodando no Rust!')").exec().unwrap();
    });
}
